src/schema/nmdc.yaml

```diff
@@ -1069,4 +1083,8 @@ slots:
 #    description: A substance within a formulation that enhances the solubility of another substance.
 #    exact_mappings:
 #      - NCIT:C176640
-
+  sample_portion:
```

Comment:
maybe sampl**ed**_portion? (other ideas: `sampled_material`, `sampled_from`).

Reply:
I like **sampled_portion**. I think that's intuitive.